src/styles/price.scss

```diff
@@ -0,0 +1,20 @@
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 16px 16px;
+  align-items: center;
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__value {
+    font-size: 16px;
```

Comment:
The font-size should be set in `rem` units to ensure better scalability and accessibility. Consider converting `16px` to `rem`.